src/classes/error-message.ts

```diff
@@ -123,8 +123,8 @@ export class ErrorMessage {
    *  error was reported
    * @returns {this} - returns the instance for chaining
    */
-  setServiceContext(service: string, version?: string) {
-    this.serviceContext.service = isString(service) ? service : 'node';
+  setServiceContext(service?: string, version?: string) {
+    this.serviceContext.service = (isString(service) ? service : 'node')!;
```

Comment:
Up to you, since you were making all these `()!` changes in this PR I would suggest doing it here. If not, can you add a TODO (maybe near the definition of `isString`?

Reply:
This sort of pattern is common across the codebase now, and so I'll make the changes in a followup.